.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
   test_macos:
     macos:
-      xcode: "13.4.1"
+      xcode: "15.3.0"
```

Comment:
macos intel resource were EOL'd I believe so I think you are going to get a sillicon resource regardless: https://discuss.circleci.com/t/macos-intel-support-deprecation-in-january-2024/48718

if that is true might want to just bump to lates 15.4.0

Reply:
sounds like we have to manually define a [resource class key](https://discuss.circleci.com/t/macos-intel-support-deprecation-in-january-2024/48718#how-to-migrate-to-m1-4) to use apple silicon.

will hokusai binary built for apple silicon work on intel? i am still on intel, as probably are some other developers. we might have to somehow do 2 builds.

Reply:
hmm, I am confused, reading that issue it sounds to me like you already cannot use non sillicon mac os executors but it seems like that is not the case and it is using the old medium gen 2 images: https://app.circleci.com/pipelines/github/artsy/hokusai/1275/workflows/6af079ea-16a5-4645-be09-a0a435f7b5d5/jobs/5483
I suppose we can just continue using this but it might start to fail at some point in the future.

Reply:
Oh June 28, 2024 I can't read. I will put it on the disaster calendar